SECURITY.md

```diff
@@ -21,8 +21,7 @@ The following versions of ClickHouse server are currently being supported with s
 | 23.5 | ❌ |
 | 23.4 | ❌ |
 | 23.3 | ✔️ |
-| 23.2 | ❌ |
```

Comment:
This makes it ambiguous.

Reply:
so, when `23.8` will remain the last supported version in 23, it's better to have all the rest of the year?

Reply:
Yes. No ambiguity.

Reply:
Ok, I'll then only improve readability